rr/ai/ai_manager.py

```diff
@@ -190,9 +190,7 @@ def __init__(
         self.on_new_prediction_cb_ = None
 
     def install_callback(self, on_new_prediction_cb_):
-        self._mutex.acquire()
         self.on_new_prediction_cb_ = on_new_prediction_cb_
-        self._mutex.release()
```

Comment:
Why was this removed?

Reply:
We are handling this in parallel this time!